### PR TITLE
centos8: install missing packages

### PR DESCRIPTION
**Pull request by VonUniGE**
Fix issue #69

### CODE REVIEW

File: centos8/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM centos:8
 LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
 LABEL name="linuxbrew/centos8"
 
-RUN yum install -y curl file make ruby sudo which \
+RUN yum install -y git curl file make ruby sudo which glibc-locale-source glibc-langpack-en \
```
Comment (sjackman) [inline centos8/Dockerfile]: Please sort these packages in alphabetical order.

**Review by sjackman — CHANGES_REQUESTED** [review]
Thank you for this fix, @VonUniGE ! Minor aesthetic quibble, then good to go.

File: centos8/Dockerfile (hunk 1 above)
Comment (VonUniGE) [reply centos8/Dockerfile]: Ok, done.

**Comment by sjackman** [comment]
Merged. Thank you for your contribution to Linuxbrew/docker, @VonUniGE!

**Pull request merged**